### ISSUE TITLE
Can't make it work

### ISSUE THREAD

**Issue by mgzenitech** [tamayika/vscode-any-lint#12]
So I have this bit set for .env files linting using poe wrapper around dotenv-linter:
```json
"any-lint.linters": [
    {
      "args": [
        "poe",
        "lint:dotenv",
        "-f",
        "${file}"
      ],
      "binPath": "poetry",
      "condition": "$.fileExtname.endsWith('.env')",
      "cwd": "${workspaceFolder}",
      "diagnostic": {
        "format": "${file}:${startLine} ${message}",
        "output": "stdout"
      },
      "name": "dotenv-linter",
      "on": [
        "change",
        "save"
      ]
    }
  ]
```

Output I get in any-lint:
```
/home/xxx/Documents/Projects/xxx/prototype-generic-vscode
poetry poe lint:dotenv -f /home/xxx/Documents/Projects/xxx/prototype-generic-vscode/.internal/.src/.env/.0.env
poetry poe lint:dotenv -f /home/xxx/Documents/Projects/xxx/prototype-generic-vscode/.internal/.src/.env/.0.env failed
.internal/.src/.env/.0.env:1 SpaceCharacter: The line has spaces around equal sign
```

Yet no `problems` are appearing in VSCode? What I'm doing wrong?

**Comment by Mart007King** [comment]
Try this:

- Ensure Extension is Installed and Enabled: Verify that the any-lint extension is installed and enabled in your VSCode setup.
- Restart VSCode: Sometimes, a simple restart can resolve issues related to configuration changes not being picked up.
- Check for Conflicts: Ensure no other linters or extensions are interfering with the output parsing.

**Comment by mgzenitech** [comment]
> Try this:
> 
>     * Ensure Extension is Installed and Enabled: Verify that the any-lint extension is installed and enabled in your VSCode setup.
> 
>     * Restart VSCode: Sometimes, a simple restart can resolve issues related to configuration changes not being picked up.
> 
>     * Check for Conflicts: Ensure no other linters or extensions are interfering with the output parsing.

Yeah, everything is enabled and installed properly as I can see the output of this extension:
![Screenshot from 2024-07-12 17-29-17](https://github.com/user-attachments/assets/e1d1162a-7a19-4503-a0c0-92cb7ea0599c)

Also tried reloading window multiple times.

Not sure how to debug this as there is no --debug switch or anything in extension settings

**Comment by tamayika** [comment]
Hi, thanks for the report.
I've fixed the problem in `0.2.2`.
`startColumn` was a required placeholder in format but I changed it optional.
If the problem still happens in the latest extension, please re-open this issue.
